pkg/tenant/db/topics_test.go

```diff
@@ -90,8 +138,21 @@ func (s *dbTestSuite) TestCreateTopic() {
 	require.NoError(err, "could not validate topic data")
 
 	s.mock.OnPut = func(ctx context.Context, in *pb.PutRequest) (*pb.PutReply, error) {
-		if len(in.Key) == 0 || len(in.Value) == 0 || in.Namespace != db.TopicNamespace {
-			return nil, status.Error(codes.FailedPrecondition, "bad Put request")
+		switch len(in.Key) {
```

Comment:
Add a story to add a "multi-get" to the trtl mock to make testing with multiple gets easier.